app/controllers/api/v1/videos_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::VideosController < ApplicationController
+  def index
+    @videos = Video.all 
+    render json: @videos.map { |user| 
+    if user.video.attached? 
+      user.as_json.merge(video_path: url_for(user.video))
+    else 
+      user.as_json only: :title
+    end }
+  end
+    
+  def create
+    @video = Video.new(create_params)  
+    if @video.save
+      render json: success_json(@video), status: :created
```

Comment:
@moizansari22 Why you are returning video details in response? Are you using it?

Reply:
no this was extra piece of code that i added . i have deleted this